src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -697,14 +697,15 @@ void uct_rc_mlx5_iface_common_query(uct_ib_iface_t *ib_iface,
     iface_attr->cap.flags        |= UCT_IFACE_FLAG_ERRHANDLE_ZCOPY_BUF |
                                     UCT_IFACE_FLAG_ERRHANDLE_REMOTE_MEM;
 
-    if (dev->pci_fadd_arg_sizes || dev->pci_cswap_arg_sizes) {
+    if (iface->super.config.fence) {
```

Comment:
this doesn't make sense when reading it (even though it's correct)
maybe use same inline function which checks dev->pci_fadd_arg_sizes and dev->pci_cswap_arg_sizes, both here, and when initializing the config.fence